ms2deepscore/spectrum_binning_fixed.py

```diff
@@ -3,17 +3,43 @@
 import numpy as np
 
 
-def create_peak_list_fixed(spectrums, class_values, d_bins, mz_min=10.0):
-    """Create list of (binned) peaks."""
+def create_peak_list_fixed(spectrums, peaks_vocab, d_bins, mz_min=10.0, peak_scaling=0.5):
```

Comment:
To me it would make more sense to output a numpy array of n_spectrums x n_bins, if you're worried about sparsity + memory you could use [scipy sparse matrices](https://docs.scipy.org/doc/scipy/reference/sparse.html). 
I think this would also make the data processing (for example scaling) a lot simpler.

Reply:
Good point, that might indeed make the processing easier and/or faster. Speed-wise its currently fine, so I would postpone this. --> #11